integration_test.go

```diff
@@ -112,6 +114,7 @@ func TestGocannonDefaultValues(t *testing.T) {
 	cfg := common.Config{
 		Duration:    &duration,
 		Connections: &connections,
+		CPUs:				 &cpus,
```

Comment:
There is a minor formatting issue here - tabs between `:` and `&` instead of spaces